mslice/plotting/plot_window/interactive_cut.py

```diff
@@ -62,6 +64,10 @@ def end_drag(self, event):
         self._canvas.mpl_disconnect(self.connect_event[0])
         self._canvas.mpl_disconnect(self.connect_event[1])
 
+    def callback(self, event):
```

Comment:
I think this name could be better - it's arguably not a callback since it doesn't get passed into a different class, and it doesn't tell us what the method does. Maybe something like `redraw_rectangle`?

Reply:
Alright